openbas-framework/src/main/java/io/openbas/executors/tanium/service/TaniumExecutorService.java

```diff
@@ -100,7 +100,9 @@ public TaniumExecutorService(
   public void run() {
     log.info("Running Tanium executor endpoints gathering...");
     List<NodeEndpoint> nodeEndpoints =
-        this.client.endpoints().getData().getEndpoints().getEdges().stream().toList();
+        this.client.endpoints().getData() != null
+            ? this.client.endpoints().getData().getEndpoints().getEdges().stream().toList()
+            : new ArrayList<>();
```

Comment:
Do we maybe want to log the error : it can help us detect a problem witth the Tanium excecutor